.github/workflows/test_contracts.yml

```
@@ -0,0 +1,31 @@
+name: Safe Core SDK Test - Contracts
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - development
+env:
+  INFURA_KEY: ${{ secrets.INFURA_KEY }}
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]
+        provider: [ethers, web3]
+        contract-version: [v1.0.0, v1.1.1, v1.2.0,v1.3.0]
```

contract-version: [v1.0.0, v1.1.1, v1.2.0,v1.3.0]
```suggestion
        contract-version: [v1.0.0, v1.1.1, v1.2.0, v1.3.0]
```